forwards_paper.tex

```diff
@@ -240,6 +241,46 @@ \subsection*{Simulation benchmarks}
 
 \plr{Insert figures/tables of performance here and describe them.}
 
+The total run times for simulations with selection are shown in Figure~\ref{fig:runtimes_selection}.  When tracking
+neutral mutations (instead of the ARG), run times increase dramatically with increasing region size ($4Nr = 4Nu$ for
```

Comment:
We resolved elsewhere to not to call it the ARG, and hence not "ARG tracking" either.  How about "pedigree recording" instead of "ARG tracking"?

Reply:
Or we have in some places also used 'ancestry tracking' which seems equally descriptive (per discussion in https://github.com/petrelharp/ftprime_ms/issues/8#issuecomment-348910693 )

Reply:
i've pushed a new commit that is basically `% s/ARG/pedigree/g` to the sections changes in this PR.

Reply:
sounds good.  at first I thought 'ancestry tracking', but then I looked at our title...

Reply:
good call. I think I'd raised the issue in #8 because our using two terms to mean the same thing is potentially confusing. I was fine with Jerome's point but new readers will probably appreciate the consistency